src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
 spring.application.name=importer
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/log_db
+spring.datasource.username=postgres
+spring.datasource.password=root
+spring.datasource.driver-class-name=org.postgresql.Driver
+
+mybatis.mapper-locations=classpath:mapper/*.xml
+
+jwt.secret="0ddf5597e02d981f8803c4cc11f015a4e52679d706edb29b595d9e466def5bcf95273a3053ab5d97ee893c23e4023b912daefaade316406a33b7685d4d223dfa"
+post.student=http://localhost:8080/students
```

Comment:
We should declare this property by using a better naming convention. The name is short and not intuitive.

Reply:
Okay